apps/src/lib/kits/maker/ui/Unit6ValidationStep.jsx

```diff
@@ -91,17 +91,38 @@ export default class Unit6ValidationStep extends Component {
                 {i18n.eligibilityReqYearConfirmInstructions()}
               </strong>
               {[
-                [Unit6Intention.NO, i18n.eligibilityYearNo()],
-                [Unit6Intention.YES_18_19, i18n.eligibilityYearYes1819()],
-                [Unit6Intention.YES_19_20, i18n.eligibilityYearYes1920()],
-                [Unit6Intention.YES_AFTER, i18n.eligibilityYearAfter()],
-                [Unit6Intention.UNSURE, i18n.eligibilityYearUnknown()]
-              ].map(([value, description]) => (
+                [
+                  Unit6Intention.NO,
+                  i18n.eligibilityYearNo(),
+                  'ui-test-id-yearNo'
+                ],
+                [
+                  Unit6Intention.YES_18_19,
+                  i18n.eligibilityYearYes1819(),
+                  'ui-test-id-1819'
+                ],
+                [
+                  Unit6Intention.YES_19_20,
+                  i18n.eligibilityYearYes1920(),
+                  'ui-test-id-1920'
+                ],
+                [
+                  Unit6Intention.YES_AFTER,
+                  i18n.eligibilityYearAfter(),
+                  'ui-test-id-after'
+                ],
+                [
+                  Unit6Intention.UNSURE,
+                  i18n.eligibilityYearUnknown(),
+                  'ui-test-id-unsure'
+                ]
+              ].map(([value, description, uiTestId]) => (
                 <label key={value}>
                   <input
                     style={styles.radio}
                     type="radio"
                     name="year"
+                    id={uiTestId}
```

Comment:
In general, it's best to avoid mixing product code and test code (lots more detail on that if you are curious to chat :) ). If there is no way to avoid mixing the two, it usually means a refactor is needed. That being said, I think you can accomplish the same by removing "input" from lines 44 and 57 of the original test suite. Thanks for working on this test upgrade!

Reply:
Another option to minimize the mingling of test and product code would be to do something like
```id={`ui-test-${value}`}``` - this should result in each input having a unique id, but we wouldn't need to add a unique id to the array of information about each item - it just gets created from the value.

Reply:
Thanks y'all for the feedback! 

Checked in person with Jessie regarding which way to go: remove lines or `id={`ui-test-${value}`}`. Concluded with removing lines. Rebased and force pushed to the branch with only the test.

Reply:
Thanks for doing that! I should have clarified my original message. You don't need to remove the entirety of line 44 and 57. You should be able to keep the asserts by just removing the word 'input' from those lines.

Reply:
Woops! Ok, I went ahead and removed only `input` from lines 44 and 57 then checked again to make sure tests are passing by running `npm run test:entry -- --entry ./test/unit/lib/kits/maker/ui/Unit6ValidationStepTest.js` and all 5 tests passed.

![all 5 tests passing](https://user-images.githubusercontent.com/17921445/56058508-66fc8680-5d16-11e9-8e0c-206c2dba648a.gif)